appliance_hardening_guide/_topics/db_ssl.md

```diff
@@ -83,21 +91,22 @@ The database appliance now accepts only connections from connecting appliances t
 
 1.  Log in as `root` to the connecting appliance.
 
-2.  Create the `.postgresql` directory in your `root` user home directory.
+3.  Copy the root certificate file from the database to the operating system's list of valid certificates:
 
-        [root@{{ site.data.product.title_short_l }}2 ~]# mkdir /root/.postgresql
+        [root@{{ site.data.product.title_short_l }}2 ~]# DATABASE=[database_appliance_fqdn]
+        [root@{{ site.data.product.title_short_l }}2 ~]# scp root@$DATABASE:/var/www/miq/vmdb/certs/root.crt /etc/pki/ca-trust/source/$DATABASE-postgres
+        [root@{{ site.data.product.title_short_l }}2 ~]# update-ca-trust extract
+        [root@{{ site.data.product.title_short_l }}2 ~]# export PGSSLROOTCERT=/etc/pki/tls/certs/ca-bundle.crt
 
-    The PostgreSQL client library, which {{ site.data.product.title_short }} also uses, looks to this directory for custom configuration files.
+    Where `[database_appliance_fqdn]` is the fully qualified domain name of the database appliance.
 
-3.  Copy the root certificate file from the database appliance to the `/root/.postgresql` directory on the connecting appliance:
+4. Test the certificate got added to the certificate database
 
-        [root@{{ site.data.product.title_short_l }}2 ~]# scp root@[database_appliance_fqdn]:/var/www/miq/vmdb/certs/root.crt /root/.postgresql/root.crt
-
-    Where `[database_appliance_fqdn]` is the fully qualified domain name of the database appliance.
+        [root@{{ site.data.product.title_short_l }}2 ~]# openssl x509 -in /etc/pki/tls/certs/ca-bundle.crt -text -noout
 
 4.  Test the connection between the connecting appliance and the database appliance by using the `psql`command:
 
-        [root@localhost ~]# psql -h [database_appliance_fqdn] -d vmdb_production
+        [root@localhost ~]# PGSSLMODE=verify-full psql -h [database_appliance_fqdn] -d vmdb_production "dbname=vmdb_production sslmode=verify-full"
```

Comment:
Is the quoted text supposed to be there?

Reply:
thnx. that is how most suggested it but I put in the `ENV` var to get better success in my environment

Reply:
hmm. translate what I typed

> thanks. made that change.